app/src/organisms/Navigation/index.tsx

```diff
@@ -25,15 +27,30 @@ import { ODD_FOCUS_VISIBLE } from '../../atoms/buttons/constants'
 
 import { useNetworkConnection } from '../../resources/networking/hooks/useNetworkConnection'
 import { getLocalRobot } from '../../redux/discovery'
+import { useFeatureFlag } from '../../redux/config'
 import { NavigationMenu } from './NavigationMenu'
 import type { ON_DEVICE_DISPLAY_PATHS } from '../../App/OnDeviceDisplayApp'
 
-const NAV_LINKS: Array<typeof ON_DEVICE_DISPLAY_PATHS[number]> = [
+let NAV_LINKS: Array<typeof ON_DEVICE_DISPLAY_PATHS[number]> = [
   '/protocols',
   '/instruments',
   '/robot-settings',
 ]
 
+// TODO(sb:7/10/24): update this wrapper to fade on both sides only when not scrolled completely to that side
+// This will be accomplished in PLAT-399
+const CarouselWrapper = styled.div`
+  display: flex;
+  flex-direction: ${DIRECTION_ROW};
+  align-items: ${ALIGN_FLEX_START};
+  width: 650px;
```

Comment:
If there is no specific reason to use `px`

```suggestion
  width: 40.625rem;
```

Reply:
and probably a bit wider (1.5rem or so) to eyeball the full width from figma. `42.25rem` works for me:

<img width="1136" alt="Screen Shot 2024-07-11 at 11 08 51 AM" src="https://github.com/Opentrons/opentrons/assets/29845468/ceac8036-0c24-4b48-a0f4-c9014caa48b9">